jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -246,8 +252,12 @@ bool ci_lazy_methodst::handle_virtual_methods_with_no_callees(
     // Add the call class to instantiated_classes and assert that it
     // didn't already exist
     const irep_idt &call_class = virtual_function_call.get(ID_C_class);
-    auto ret_class = instantiated_classes.insert(call_class);
-    CHECK_RETURN(ret_class.second);
+    bool added_class = instantiated_classes.count(call_class) == 0;
```

Comment:
:hammer_and_pick: `added_class` is not a very meaningful name for a boolean. Maybe something like `is_newly_found_class`, or for the negation, `call_class_already_added`, would be clearer.